ydb/core/wrappers/s3_storage_config.cpp

```diff
@@ -1,81 +1,15 @@
 #include "s3_storage.h"
 #include "s3_storage_config.h"
 
-#include <contrib/libs/aws-sdk-cpp/aws-cpp-sdk-core/include/aws/core/internal/AWSHttpResourceClient.h>
-#include <contrib/libs/aws-sdk-cpp/aws-cpp-sdk-core/include/aws/core/utils/stream/PreallocatedStreamBuf.h>
-#include <contrib/libs/aws-sdk-cpp/aws-cpp-sdk-core/include/aws/core/utils/stream/ResponseStream.h>
-#include <contrib/libs/aws-sdk-cpp/aws-cpp-sdk-core/include/aws/core/Aws.h>
 #include <contrib/libs/aws-sdk-cpp/aws-cpp-sdk-core/include/aws/core/utils/threading/Executor.h>
 
-#include <contrib/libs/curl/include/curl/curl.h>
-#include <ydb/library/actors/core/actorsystem.h>
-#include <ydb/library/actors/core/log.h>
 #include <util/string/cast.h>
 
 #ifndef KIKIMR_DISABLE_S3_OPS
 namespace NKikimr::NWrappers::NExternalStorage {
 
-using namespace Aws;
-using namespace Aws::Auth;
-using namespace Aws::Client;
-using namespace Aws::S3;
-using namespace Aws::S3::Model;
-using namespace Aws::Utils::Stream;
-
 namespace {
 
-struct TCurlInitializer {
-    TCurlInitializer() {
-        curl_global_init(CURL_GLOBAL_ALL);
-    }
-
-    ~TCurlInitializer() {
-        curl_global_cleanup();
-    }
-};
-
-struct TApiInitializer {
-    TApiInitializer() {
-        Options.httpOptions.initAndCleanupCurl = false;
-        InitAPI(Options);
-
-        Internal::CleanupEC2MetadataClient(); // speeds up config construction
```

Comment:
This function is from `Internal` namespace. I don't think we should call it. Moreover, `TApiInitializer` would be initialized only once per `ydbd` application start, so its performance should not be an issue.